### PR TITLE
Add new groups system to comments

### PR DESCRIPTION
**Pull request by Tanza3D**
Still need to add this on Profiles, but should be working across all comments. Can't test snapshots because of #45

### CODE REVIEW

**Comment by Tanza3D** [comment]
needs db update on server, will do after push

File: global/php/functions.php [hunk 1]
```diff
@@ -111,6 +113,7 @@
                                     echo "0";
                                 } ?>;
         const roles = <?php echo json_encode($roles); ?>;
+        const userGroups = <?php echo json_encode($groups); ?>;
```
Comment (EXtremeExploit) [inline global/php/functions.php:116]: userGroups may indicate the groups of  the user not the groups that are available to users :thinking: i think a better name would be just `groups`

**Review by EXtremeExploit — APPROVED** [review]
lgtm except for some variable name but its good i think

File: global/php/functions.php (hunk 1 above)
Comment (Tanza3D) [reply global/php/functions.php:116]: i couldn't call it `groups` as that had conflicts on other apps annoyingly, snapshots completely broke with it

**Pull request merged**